smoke-test/pom.xml

```diff
@@ -13,13 +13,20 @@
     <camelVersion>2.19.1</camelVersion>
     <surefireVersion>2.18</surefireVersion>
     <failsafeVersion>2.22.2</failsafeVersion>
+    <groovyVersion>2.5.16</groovyVersion>
     <frontendPluginVersion>0.0.29</frontendPluginVersion>
     <jackson2Version>2.12.3</jackson2Version>
     <jersey.version>2.36</jersey.version>
     <karafVersion>4.3.6</karafVersion>
-    <selenium.version>3.141.59</selenium.version>
-    <testcontainers.version>1.15.3</testcontainers.version>
+    <selenium.version>4.6.0</selenium.version>
+    <!-- seleniarm.version needs to match a date stamped release version from the docker-seleniarm releases page
+         for the same version of selenium specified in selenium.version above.
+         https://github.com/seleniumhq-community/docker-seleniarm/releases -->
+    <seleniarm.version>4.6.0-20221104</seleniarm.version>
```

Comment:
should you make this `<seleniarm.version>${selenium.version}-20221104</seleniarm.version>` so it breaks gracefully if one gets changed an not the other?

Reply:
Eh, it's kinda six of one, half dozen of the other. The most important thing for it to match is a release in https://github.com/seleniumhq-community/docker-seleniarm/releases and it's a good idea (but not strictly mandatory, AFAICT) if that doesn't match the Selenium Java library version.